design_docs/rsyslog_inputs_outputs_flows.md

```diff
@@ -40,7 +40,7 @@ logging_outputs
   - name: files_output
     type: files
     severity: info
-    exclude: authpriv.none;auth.none;cron.none;mail.none
+    exclude: ;authpriv.none;auth.none;cron.none;mail.none
```

Comment:
see above

Reply:
I think if you fix this to use the `list` format, this PR is good to go

Reply:
Thank you, @richm. I've fixed the line 43, squashed commits and pushed.